crates/oci-tar-builder/src/lib.rs

```diff
@@ -44,10 +79,12 @@ struct DockerManifest {
 
 pub const WASM_LAYER_MEDIA_TYPE: &str =
     "application/vnd.bytecodealliance.wasm.component.layer.v0+wasm";
+pub const WASM_ARTIFACT_LAYER: &str = "application/wasm";
```

Comment:
Heads up that both of these values are available as consts in `oci_wasm`

Reply:
woops, that was before I realized I wanted to use the `oci_wasm` package. They weren't referenced any more.  Thanks!